chemicaltoolbox/mordred/.shed.yml

```diff
@@ -0,0 +1,17 @@
+categories:
+- Computational chemistry
+description: Mordred is a molecular descriptor calculator.
+long_description: |
+  Mordred is a descriptor-calculation software application that can calculate more than 1800 two- and three-dimensional descriptors.
+  
+  Home page: https://github.com/mordred-descriptor/mordred
+  License: BSD
+
+homepage_url: https://github.com/mordred-descriptor/mordred
+auto_tool_repositories:
```

Comment:
are you planning to add more tools in this repo? If not these auto_tool_repositories are not needed as you only have one tool